include/libtorrent/torrent_info.hpp

```diff
@@ -268,6 +268,7 @@ namespace libtorrent {
 		// number. For more information, see announce_entry_.
 		void add_tracker(std::string const& url, int tier = 0);
 		std::vector<announce_entry> const& trackers() const { return m_urls; }
+		void clear_trackers();
```

Comment:
could you include `clear_trackers()` in the documentation above as well?

Reply:
Ok